README.md

```diff
@@ -312,6 +333,22 @@ val forage = ForageTerminalSDK(posTerminalId)
 
 You can then use the instance to call methods that perform payment operations, as demonstrated in the following section.
 
+#### Initialize a `ForageTerminalSDK` instance
+⚠️ **Before you can execute `ForageTerminalSDK` methods, you must call the `init` function on the `ForageTerminalSDK` instance.**
+
+`init` returns the original instance.
+
+```kotlin
+val forage = ForageTerminalSDK(posTerminalId)
+
+forage.init(sessionToken)
+```
+
+##### `init` parameters
+- `sessionToken`: A short-lived token that authenticates front-end requests to Forage. To create one, send a server-side request from your backend to the [`/session_token/`](https://docs.joinforage.app/reference/create-session-token) endpoint.
+
+⚠️ **`init` can take up to 10 seconds to run.** While it doesn’t often take that much time, account for the potential delay in your app’s UX design.
```

Comment:
thanks for including this note! Basically the main consideration that we want merchants to be aware of